distributions/openhab/src/main/resources/bin/update.ps1

```diff
@@ -1,4 +1,27 @@
-#Requires -Version 5.0
+#Requires -Version 4.0
+function DownloadFiles {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string]$DownloadSource,
+        [Parameter(Mandatory=$true)]
+        [string]$Outputfile
+    )
+    $client = New-Object System.Net.WebClient
+    try {
+        $client.DownloadFileAsync($DownloadLocation,$Outputfile)
+        while ($client.IsBusy)
+        {
+            Write-Host "Downloading file..."
```

Comment:
I used BITS in mine as well (and had cool progress scripting) but like you said, the cloudbees has an issue with that. I think it may have something to do with http 1.1. Since I was looking into downgrading to PS 2.0 I stated looking here. I could revert to Invoke-WebRequest for both. (I wish BITS worked for CB)